exaudfclient/base/javacontainer/script_options/converter.h

```diff
@@ -19,15 +14,13 @@ class Converter {
     public:
         Converter();
     
-        void convertExternalJar(const std::string & value);
+        virtual void convertExternalJar(const std::string & value) = 0;
 
         void convertScriptClassName(const std::string & value);
 
         void convertJvmOption(const std::string & value);
 
-        const std::set<std::string> & getJarPaths() const {
-            return m_jarPaths;
-        }
+        virtual const std::set<std::string> & getJarPaths() const = 0;
```

Comment:
generally I think it is helpful for the reader to group virtual and non virtual functions/methods together

Reply:
Done